src/components/common/EffectiveDateTime.vue

```diff
@@ -358,12 +357,6 @@ export default class EffectiveDateTime extends Mixins(DateMixin) {
     }
   }
 
-  @Watch('datePicker')
-  onDatePickerChanged (val: string): void {
-    this.dateText = val
-    // the watcher for dateText will fire next
-  }
```

Comment:
`dateText` was essentially unused because it was initialized as '' (line 173) and possibly assigned to '' (line 402).

I tested this component and it all works correctly.